src/Sparrow/DefaultFormat.cs

```diff
@@ -2,6 +2,7 @@ namespace Sparrow
 {
     internal static class DefaultFormat
     {
+        public static readonly string TimeOnlyAndDateOnlyFormatToWrite = "O";
```

Comment:
1. What's the difference between "O" vs "o"?
2. I'm not sure if having single field for both types it good. Maybe introduce:

```
public static readonly string TimeOnlyFormatToWrite = "O";
public static readonly string DateOnlyFormatToWrite = "O";
```

Reply:
1. Can we put "o" then? Consistency with below:

`public static readonly string DateTimeOffsetFormatsToWrite = "o";`

Reply:
done